doc/man1/openssl-verification-options.pod

```diff
@@ -151,20 +173,18 @@ The first step is to check that each certificate is well-formed.
 Part of these checks are enabled only if the B<-x509_strict> option is given.
 
 The second step is to check the extensions of every untrusted certificate
-for consistency with the supplied purpose.
-If the B<-purpose> option is not given then no such checks are done
-except for SSL/TLS connection setup,
-where by default C<sslserver> or C<sslclient>, are checked.
-The target or "leaf" certificate, as well as any other untrusted certificates,
-must have extensions compatible with the specified purpose.
+for consistency with the given required purpose.
+If the B<-purpose> option is not given
+then no such checks are done except for SSL/TLS connection setup,
+where by default the purpose C<sslserver> or C<sslclient> is checked.
```

Comment:
Note, this uses C&lt;sslserver&gt; and C&lt;sslient&gt;, while I suggested B&lt;sslserver&gt;, ... Perhaps C... is more correct.

Reply:
I believe so.